errors/B0003.md

```diff
@@ -11,98 +11,52 @@ fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
         .add_systems(Startup, setup)
-        .add_systems(Update, despawning)
-        .add_systems(Update, use_entity.after(despawning))
+        .add_systems(Update, (use_0_and_despawn_1, use_1_and_despawn_0))
         .run();
 }
 
 #[derive(Resource)]
-struct MyEntity(Entity);
+struct MyEntities([Entity; 2]);
 
 #[derive(Component)]
 struct Hello;
 
 fn setup(mut commands: Commands) {
-    let entity = commands.spawn_empty().id();
-    commands.insert_resource(MyEntity(entity));
+    let entity1 = commands.spawn_empty().id();
+    let entity2 = commands.spawn_empty().id();
+    commands.insert_resource(MyEntities([entity1, entity2]));
 }
 
-fn despawning(mut commands: Commands, entity: Option<Res<MyEntity>>) {
-    if let Some(my_entity) = entity {
-        commands.entity(my_entity.0).despawn();
-        commands.remove_resource::<MyEntity>();
-    }
+fn use_0_and_despawn_1(mut commands: Commands, my_entities: Res<MyEntities>) {
+    commands.entity(my_entities.0[0]).insert(Hello);
+    commands.entity(my_entities.0[1]).despawn();
 }
 
-fn use_entity(mut commands: Commands, entity: Option<Res<MyEntity>>) {
-    if let Some(my_entity) = entity {
-        commands.entity(my_entity.0).insert(Hello);
-    }
+fn use_1_and_despawn_0(mut commands: Commands, my_entities: Res<MyEntities>) {
+    commands.entity(my_entities.0[1]).insert(Hello);
+    commands.entity(my_entities.0[0]).despawn();
 }
 ```
 
-This will panic, as system `use_entity` is executed after system `despawning`. Without the system ordering specified here, the ordering would be random and this code would panic half the time.
+This will panic, as the system that is executed first will despawn the entity used by the second.
 
-The default panic message is telling you the entity id (`1v0`) and the command that failed (adding a component `Hello`):
+The default panic message is telling you the entity id (`2v0`), the command that failed (adding a component `Hello`) and the system from which it originated (`use_1_and_despawn_0`):
```

Comment:
This grammar doesn't make sense.
```suggestion
The default panic message is telling you the entity (`2v0`) doesn't exist, and the command that failed. (adding a component `Hello`) and the system from which it originated (`use_1_and_despawn_0`):
```
Additionally there are 2 cases for what entity doesn't exist (either `1v0` or `2v0` based on system ordering)